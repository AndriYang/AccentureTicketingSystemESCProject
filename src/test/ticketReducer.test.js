import ticketReducer from '../store/reducers/ticketReducer'

describe('With snapshots', () => {
  it('should return initial state', () =>{
    expect(ticketReducer(undefined, {})).toMatchSnapshot();
  })

  it('should handle CREATE_PROJECT', ()=>{
    const action = {
      type: 'CREATE_PROJECT',
    }
    expect(ticketReducer(undefined, action)).toMatchSnapshot();
  })

  it('should handle CREATE_PROJECT_ERROR', ()=>{
    const action = {
      type: 'CREATE_PROJECT_ERROR',
    }
    expect(ticketReducer(undefined, action)).toMatchSnapshot();
  })
})
