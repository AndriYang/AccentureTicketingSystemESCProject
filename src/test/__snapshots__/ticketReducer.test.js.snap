// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots should handle CREATE_PROJECT 1`] = `
Object {
  "projects": Array [
    Object {
      "content": "blah blah blah",
      "id": "1",
      "title": "help me find peach",
    },
    Object {
      "content": "blah blah blah",
      "id": "2",
      "title": "collect all the start",
    },
    Object {
      "content": "blah blah blah",
      "id": "3",
      "title": "egg hunt with me",
    },
  ],
}
`;

exports[`With snapshots should handle CREATE_PROJECT_ERROR 1`] = `
Object {
  "projects": Array [
    Object {
      "content": "blah blah blah",
      "id": "1",
      "title": "help me find peach",
    },
    Object {
      "content": "blah blah blah",
      "id": "2",
      "title": "collect all the start",
    },
    Object {
      "content": "blah blah blah",
      "id": "3",
      "title": "egg hunt with me",
    },
  ],
}
`;

exports[`With snapshots should return initial state 1`] = `
Object {
  "projects": Array [
    Object {
      "content": "blah blah blah",
      "id": "1",
      "title": "help me find peach",
    },
    Object {
      "content": "blah blah blah",
      "id": "2",
      "title": "collect all the start",
    },
    Object {
      "content": "blah blah blah",
      "id": "3",
      "title": "egg hunt with me",
    },
  ],
}
`;
